packages/block-library/src/embed/index.js

```diff
@@ -237,7 +237,7 @@ export function getEmbedEdit( title, icon ) {
 				<figure className={ classnames( className, 'wp-block-embed', { 'is-video': 'video' === type } ) }>
 					{ controls }
 					{ ( cannotPreview ) ? (
-						<Placeholder icon={ icon } label={ __( 'Embed URL' ) }>
+						<Placeholder icon={ <BlockIcon icon={ icon } showColors /> } label={ label } className="wp-block-embed">
```

Comment:
Why are we duplicating the `class` attribute here (it's already on the `figure` above)? cc @notnownikki @jorgefilipecosta